tests/conftest.py

```diff
@@ -16,3 +17,13 @@ def restore_resolvers() -> Any:
     state = copy.deepcopy(BaseContainer._resolvers)
     yield
     BaseContainer._resolvers = state
+
+
+@pytest.fixture(scope="function")
+def register_identity(restore_resolvers: Any) -> Any:
+    """
+    A fixture to register the common `identity` resolver.
+    It depends on `restore_resolvers` to make it easier and safer to use.
+    """
+    OmegaConf.register_new_resolver("identity", lambda x: x)
+    yield
```

Comment:
does this play well with another fixture registering another function?

Reply:
As far as I can think of, yes: assuming a similar pattern (depending on `restore_resolvers`, we will just copy & restore resolvers multiple times, but everything should work fine.

If we don't depend on `restore_resolvers` then users need to make sure to always put `restore_resolvers` first, which is more cumbersome and error prone.

Reply:
> assuming a similar pattern (depending on `restore_resolvers`, we will just copy & restore resolvers multiple times, but everything should work fine.

Actually this is wrong, `restore_resolvers` is called only once (which is even better).

Reply:
Another thing you can try is to unregister after the yield. this way you don't need to depend on restore_resolvers.
Either way, this is good.

Reply:
There's no unregister API right now => resolving